effects/stage_effect.py

```diff
@@ -0,0 +1,55 @@
+from game import Game
+from effects.effect import Effect
+from citizens.citizen import Citizen
+from citizens.player import Player
+import user_interaction
+import utils
+
+
+class StageEffect(Effect):
+    def __init__(self, name: str, game: Game, creator: Citizen):
+        Effect.__init__(self, name, game, creator)
+
+    def activate(self, target_number):
+        target_number = user_interaction.read_index('config.type_target')
+        while(not self.validate(target_number)):
+            user_interaction.show_active_instant("config.type_error")
+            target_number = user_interaction.read_index("config.type_target")
+
+        self.target = self.game.citizens[target_number - 1]
+
+    def resolve(self):
+        self.creator._staging_available = False
+        # self.creator.allowed_actions.remove(CARD_USAGE)
```

Comment:
please, use instead of CARD_USAGE -> ActionType.CARD_USAGE

Reply:
Done